resources/views/pages/saved-items.blade.php

```diff
@@ -0,0 +1,162 @@
+                        @foreach($tips as $tip)
+                        @if($tip->id == $item->item_id && $item->folder == null)
+                        <h4>{{date('d-m-Y', strtotime($item->created_at))}}</h4>
+                        @card
+                        <h5>{{ __('tips.personal-tip') }}</h5>
+                        <div class="alert" style="background-color: #00A1E2; color: white; margin-left:2px; margin-bottom: 10px"
+                             role="alert">
+                             <a onclick="chooseItem({{ $item->sli_id }})" data-target="#addItemModel" data-toggle="modal"><span class="glyphicon glyphicon-plus add-tip" aria-hidden="true"></span></a>
+                            <h4 class="tip-title">{{ __('tips.personal-tip') }}</h4>
+                            <p>{!! nl2br($tip->tipText) !!}</p>
+                        </div>
+                        @endcard
+                        @endif
+                    @endforeach
+                    @endif
+                    @endforeach
+
+                @endcard
+
+            </div>
+            <div class="col-md-6">
+            @card
+                <h2 class="maps">{{ __('folder.folders') }}</h2> <a data-target="#addFolderModel" data-toggle="modal"><span class="glyphicon glyphicon-plus add-collapse" aria-hidden="true"></span></a>
+
+                @foreach($folders as $folder)
+                    <div class="card card-body">
+                        <button class="btn-collapse" data-toggle="collapse" href="#{{$folder->folder_id}}" role="button" aria-expanded="false" aria-controls="collapseExample">
+                        <strong>{{ $folder->title }}</strong>
+                        </a>
+                        <div class="collapse" id="{{$folder->folder_id}}">
+                            <i>{{date('d-m-Y', strtotime($folder->created_at))}}</i>
+                            <br>
+                            {{ $folder->description }}
+
+                            @foreach($sli as $item)
+                                @if($item->category == 'tip')
+                                    @foreach($tips as $tip)
+                                        @if($tip->id == $item->item_id && $item->folder == $folder->folder_id)
+                                            <div class="alert" style="background-color: #00A1E2; color: white; margin-left:2px; margin-bottom: 10px"
+                                                role="alert">
+                                                <h4 class="tip-title">{{ __('tips.personal-tip') }}</h4>
+                                                <p>{!! nl2br($tip->tipText) !!}</p>
+                                            </div>
+                                         @endif
+                                    @endforeach
+                                @endif
+                            @endforeach
+                        </div>
+                    </div>   
+                 @endforeach
+
```

Comment:
The nested foreach loops here are something we should not want. 
Try to create a map/associative array where the key is the ID of the item. So a simple lookup can be done instead of using `@foreach($tips as $tip)` every time. Also don't forget to use strict comparison checking: `===` instead of `==`.